packages/setup-remote-server/package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "@wearerequired/setup-remote-server",
+  "version": "0.1.0-alpha.0",
+  "description": "Setup environment on remote server for deployment.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/wearerequired/js.git"
+  },
+  "keywords": [
+    "cli",
+    "remote",
+    "sever",
```

I suppose this should be 
```suggestion
    "server",
```